python/MooseDocs/extensions/MooseCommonExtension.py

```diff
@@ -49,8 +49,8 @@ def getSettings(self, settings_line):
       SETTINGS_RE = re.compile("([^\s=]+)=(.*?)(?=(?:\s[^\s=]+=|$))")
       matches = SETTINGS_RE.findall(settings_line.strip())
 
-      if not matches:
-        return {}
+      if len(matches) == 0:
+          return self._settings, {}
```

Comment:
I think I need to scrutinize this a bit more. This was removed, while trying to figure out the markdown extension RE routine not matching correctly when no attributes were applied. Do we need this? Was this just here to speed things up?